jdi-light/src/main/java/com/epam/jdi/light/logger/JDILogger.java

```diff
@@ -76,26 +83,30 @@ public void logOff(JAction action) {
     public <T> T logOff(JFunc<T> func) {
         LogLevels tempLevel = logLevel.get();
         if (logLevel.get() == OFF) {
-            try { return func.invoke();
+            try {
+                return func.invoke();
             } catch (Exception ex) {
                 throw new RuntimeException(ex);
             }
         }
         logLevel.set(OFF);
         T result;
-        try{ result = func.invoke(); }
-        catch (Exception ex) {
+        try {
+            result = func.invoke();
+        } catch (Exception ex) {
             throw new RuntimeException(ex);
         }
         logLevel.set(tempLevel);
         return result;
     }
-    private String name;
-    private Logger logger;
-    private List<Long> multiThread = new ArrayList<>();
+
+    private final String name;
+    private final Logger logger;
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Fields should be declared at the top of the class, before any method declarations, constructors, initializers or inner classes.](https://app.codacy.com/manual/42_/jdi-light/pullRequest?prid=5482899)